### PR TITLE
EID-1823 Update urls for multi-country build release

### PR DESCRIPTION
**Pull request by idrop**
Reverts alphagov/verify-proxy-node#375, reapplying PR https://github.com/alphagov/verify-proxy-node/pull/371

**TL;DR**
set Gateway hostname to https://proxy-node.eidas.test.london.verify.govsvc.uk/
set Stub Connector hostname to https://stub-connector.eidas.test.london.verify.govsvc.uk/

### CODE REVIEW

**Review by CharlesIC — COMMENTED** [review]
.

File: ci/build/build-pipeline.yaml [hunk 1]
```diff
@@ -177,13 +177,13 @@ spec:
 
     groups:
       - name: deployment
-        jobs: [build-vsp-image, build-proxy-node, package, test, release]
```
Comment (CharlesIC) [inline ci/build/build-pipeline.yaml]: Discard these changes - it should be safe to deploy

File: ci/build/build-pipeline.yaml [hunk 2]
```diff
@@ -501,154 +501,154 @@ spec:
...
+#              echo "Extracting chart package from the release image..."
+#              cp -r /proxy-node-chart-package/* chart/
+#
+#      - put: release
+#        params:
+#          name: chart/tag
+#          tag: chart/tag
+#          commitish: chart/ref
+#          globs:
+#          - chart/*.tgz*
```
Comment (CharlesIC) [inline ci/build/build-pipeline.yaml]: Discard these changes - it should be safe to deploy

File: ci/sandbox/build-pipeline.yaml [hunk 3]
```diff
@@ -600,8 +599,8 @@ spec:
         config:
           platform: linux
           params:
-            PROXY_NODE_URL: "https://test-proxy-node.((cluster.domain))"
-            STUB_CONNECTOR_URL: "https://test-connector.((cluster.domain))"
+            PROXY_NODE_URL: "https://eidas-proxy-node-test.((cluster.domain))"
+            STUB_CONNECTOR_URL: "https://eidas-stub-connector-test.((cluster.domain))"
```
Comment (CharlesIC) [inline ci/sandbox/build-pipeline.yaml]: I'd revert these changes - this can cause issues in sandbox. We don't need to change the sandbox URLs.

File: stub-connector/src/test/resources/metadata/test-stub-connector-metadata.xml [hunk 4]
```diff
@@ -19,9 +19,9 @@
     </md:Extensions>
     <md:IDPSSODescriptor WantAuthnRequestsSigned="true" protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
         <md:NameIDFormat xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata">urn:oasis:names:tc:SAML:2.0:nameid-format:persistent</md:NameIDFormat>
-        <md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://test-proxy-node.london.verify.govsvc.uk/SAML2/SSO/POST" 
+        <md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://proxy-node.invalid/SAML2/SSO/POST"
```
Comment (CharlesIC) [inline stub-connector/src/test/resources/metadata/test-stub-connector-metadata.xml]: Not that it matters but this file is `test-stub-connector-metadata.xml` to this should be `stub-connector.invalid`

File: stub-connector/src/test/resources/metadata/test-stub-connector-metadata.xml [hunk 5]
```diff
@@ -19,9 +19,9 @@
...
             xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata"/>
-        <md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="https://test-proxy-node.london.verify.govsvc.uk/Redirect" 
+        <md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="https://proxy-node.invalid/Redirect"
```
Comment (CharlesIC) [inline stub-connector/src/test/resources/metadata/test-stub-connector-metadata.xml]: Same - this should be `stub-connector.invalid` (but it doesn't really matter)

**Review by Wynndow — APPROVED** [review]

**Review by CharlesIC — APPROVED** [review]
